src/views/Imprint.vue

```diff
@@ -1,13 +1,13 @@
 <template>
-  <div>
+  <div id="imprint-view">
+    <main-header :headerTitle="'Impressum'"></main-header>
     <b-container>
       <b-row>
         <b-col class="d-flex justify-content-end custom-button mb-n5 mr-2 mt-3">
           <b-button pill to="/options"><vue-fontawesome  icon="times" color="white" size="1"></vue-fontawesome></b-button>
```

Comment:
Kann man den Button nicht auslagern, wenn der auf so vielen Seiten wieder verwendet wird?

Reply:
Den würde ich in einem späteren PR eh ersetzen wollen: https://github.com/Sybit-Education/Coding-Camp-2020/issues/69